ovs-p4rt/sidecar/ovsp4rt.cc

```diff
@@ -48,6 +48,10 @@ std::string EncodeByteValue(int arg_count...) {
   return byte_value;
 }
 
+static inline std::string EncodeVniValue(uint16_t vni) {
+  return EncodeByteValue(3, 0, (vni >> 8) & 0xFF, vni & 0xFF);
```

Comment:
So, this restores to previous encoding byte order.

We will still have the issue of IPv6 not working for over 1024 values, but thats a smaller problem than what we have currently. We can take the IPv6 issue on later

Reply:
Small batches. Incremental improvement. It's one of the core principles of modern software development.